src/pages/register.js

```diff
@@ -73,170 +81,160 @@ const Register = (props) => {
-              id="state"
-              className="w-full p-2.5 ml-2 bg-transparent outline-none"
-              onChange={onChange}
-              required
-            />
-          </div>
+
+          <input
+            name="state"
+            type="text"
+            placeholder="state"
+            id="state"
+            className="w-full mt-2 px-3 py-2 text-gray-500 bg-transparent outline-none border focus:border-indigo-600 shadow-sm rounded-lg"
+            onChange={onChange}
+            required
+          />
         </div>
         <div className="max-w-md px-4 mx-auto mt-12">
           <label htmlFor="occupation" className="block py-2 text-gray-500">
             Occupation
           </label>
-          <div className="flex items-center text-gray-400 border rounded-md">
-            <div className="px-3 py-2.5 rounded-l-md bg-gray-50 border-r"></div>
-            <input
-              name="occupation"
-              type="text"
-              placeholder="occupation"
-              id="occupation"
-              className="w-full p-2.5 ml-2 bg-transparent outline-none"
-              onChange={onChange}
-              required
-            />
-          </div>
+
+          <input
+            name="occupation"
+            type="text"
+            placeholder="occupation"
+            id="occupation"
+            className="w-full mt-2 px-3 py-2 text-gray-500 bg-transparent outline-none border focus:border-indigo-600 shadow-sm rounded-lg"
+            onChange={onChange}
+            required
+          />
         </div>
         <div className="max-w-md px-4 mx-auto mt-12">
           <label htmlFor="state" className="block py-2 text-gray-500">
             Bio
           </label>
-          <div className="flex items-center text-gray-400 border rounded-md">
-            <div className="px-3 py-2.5 rounded-l-md bg-gray-50 border-r"></div>
-            <textarea
-              name="bio"
-              type="text"
-              placeholder="bio"
-              id="state"
-              rows="4"
-              col="50"
-              className="w-full p-2.5 ml-2 bg-transparent outline-none"
-              onChange={onChange}
-              required
-            />
-          </div>
+
+          <textarea
+            name="bio"
+            type="text"
+            placeholder="bio"
+            id="state"
+            rows="4"
+            col="50"
+            className="w-full mt-2 px-3 py-2 text-gray-500 bg-transparent outline-none border focus:border-indigo-600 shadow-sm rounded-lg"
+            onChange={onChange}
+            required
+          />
         </div>
+
         <div className="flex justify-center">
-          <button className="px-5 py-3 text-white duration-150 bg-indigo-600 rounded-lg hover:bg-indigo-700 active:shadow-lg">
-            Signup
+          <button className="mt-5 px-6 py-2 text-white duration-150 bg-indigo-600 rounded-lg hover:bg-indigo-700 active:shadow-lg">
```

Comment:
Let's scrap the all caps and use "Remove" instead

Reply:
Remove what'?